src/Couchbase.Lite.Shared/API/Document/ArrayObject.cs

```diff
@@ -130,7 +130,6 @@ internal MCollection ToMCollection()
 
         private static MValue Get(FleeceMutableArray array, int index, IThreadSafety? threadSafety = null)
         {
-            array.Context.CheckDisposed();
```

Comment:
No longer needed since the underlying Get call will verify this, making this call redundant